### DISCUSSION TITLE
Using Precise wall and inner/outer/inner printing order

### DISCUSSION THREAD

**Discussion by matulekpl** [OrcaSlicer/OrcaSlicer#4697] [General]
Hello,
As manual says this precise wall option will only work with inner/outer option. 
But what with inner/outer/inner? Should it work also with it?

P.s. What will be better? using precise wall and inner/outer or using outer/inner?

**Comment by igiannakas** [comment]
Precise wall is useful when the middle perimeter is printed before the outer perimeter, as it adjusts the spacing between these two lines to allow for more space for the external perimeter to expand into and not deform from any irregularities in the internal perimeters.

Therefore, it is not needed when printing inner-outer-inner or outer-inner as the first internal perimeter is printed after the external, hence has no influence on its appearance.

In terms of wall quality inner-outer-inner will give you the best results, followed by outer - inner (which does not have any material benefit in the vast majority of cases) followed by inner-outer. 

Inner-Outer will give you better very sharp overhangs (i.e. over 45-50 degrees) where the inner-outer-inner or outer inner modes struggle as they dont have a perimeter right next to the external perimeter to anchor it to.
